python312Packages.deepdiff: 7.0.1 -> 8.0.1

doronbehar opened the pull request:
## Description of changes


## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

doronbehar committed:
python312Packages.deepdiff: 7.0.1 -> 8.0.0

Diff: https://github.com/seperman/deepdiff/compare/refs/tags/7.0.1...8.0.0

Changelog: https://github.com/seperman/deepdiff/releases/tag/8.0.0

doronbehar committed:
python312Packages.deepdiff: modernize

doronbehar committed:
python312Packages.deepdiff: adopt by doronbehar

doronbehar committed:
python312Packages.orderly-set: init at 5.2.2

doronbehar commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 348567`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>azure-cli-extensions.confcom</li>
    <li>azure-cli-extensions.confcom.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 115 packages failed to build:</summary>
  <ul>
    <li>ansible (ansible_2_17 ,python312Packages.ansible-core)</li>
    <li>ansible-later</li>
    <li>ansible-later.dist</li>
    <li>ansible-lint</li>
    <li>ansible-lint.dist</li>
    <li>ansible-navigator</li>
    <li>ansible-navigator.dist</li>
    <li>ansible.dist (ansible_2_17.dist ,python312Packages.ansible-core.dist)</li>
    <li>ansible_2_15</li>
    <li>ansible_2_15.dist</li>
    <li>ansible_2_16</li>
    <li>ansible_2_16.dist</li>
    <li>dbt (python312Packages.dbt-core)</li>
    <li>dbt.dist (python312Packages.dbt-core.dist)</li>
    <li>home-assistant-component-tests.apple_tv</li>
    <li>home-assistant-component-tests.litterrobot</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>molecule (python312Packages.molecule)</li>
    <li>molecule.dist (python312Packages.molecule.dist)</li>
    <li>napalm</li>
    <li>napalm.dist</li>
    <li>peering-manager</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python311Packages.ansible</li>
    <li>python311Packages.ansible-compat</li>
    <li>python311Packages.ansible-compat.dist</li>
    <li>python311Packages.ansible-core</li>
    <li>python311Packages.ansible-core.dist</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-kernel.dist</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.ansible-runner.dist</li>
    <li>python311Packages.ansible-vault-rw</li>
    <li>python311Packages.ansible-vault-rw.dist</li>
    <li>python311Packages.ansible.dist</li>
    <li>python311Packages.dbt-adapters</li>
    <li>python311Packages.dbt-adapters.dist</li>
    <li>python311Packages.dbt-bigquery</li>
    <li>python311Packages.dbt-bigquery.dist</li>
    <li>python311Packages.dbt-common</li>
    <li>python311Packages.dbt-common.dist</li>
    <li>python311Packages.dbt-core</li>
    <li>python311Packages.dbt-core.dist</li>
    <li>python311Packages.dbt-postgres</li>
    <li>python311Packages.dbt-postgres.dist</li>
    <li>python311Packages.dbt-redshift</li>
    <li>python311Packages.dbt-redshift.dist</li>
    <li>python311Packages.dbt-snowflake</li>
    <li>python311Packages.dbt-snowflake.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.molecule</li>
    <li>python311Packages.molecule.dist</li>
    <li>python311Packages.napalm</li>
    <li>python311Packages.napalm-hp-procurve</li>
    <li>python311Packages.napalm-hp-procurve.dist</li>
    <li>python311Packages.napalm.dist</li>
    <li>python311Packages.pyatv</li>
    <li>python311Packages.pyatv.dist</li>
    <li>python311Packages.pylitterbot</li>
    <li>python311Packages.pylitterbot.dist</li>
    <li>python311Packages.pytest-ansible</li>
    <li>python311Packages.pytest-ansible.dist</li>
    <li>python311Packages.pytest-testinfra</li>
    <li>python311Packages.pytest-testinfra.dist</li>
    <li>python311Packages.ttp</li>
    <li>python311Packages.ttp.dist</li>
    <li>python312Packages.ansible</li>
    <li>python312Packages.ansible-compat</li>
    <li>python312Packages.ansible-compat.dist</li>
    <li>python312Packages.ansible-kernel</li>
    <li>python312Packages.ansible-kernel.dist</li>
    <li>python312Packages.ansible-runner</li>
    <li>python312Packages.ansible-runner.dist</li>
    <li>python312Packages.ansible-vault-rw</li>
    <li>python312Packages.ansible-vault-rw.dist</li>
    <li>python312Packages.ansible.dist</li>
    <li>python312Packages.dbt-adapters</li>
    <li>python312Packages.dbt-adapters.dist</li>
    <li>python312Packages.dbt-bigquery</li>
    <li>python312Packages.dbt-bigquery.dist</li>
    <li>python312Packages.dbt-common</li>
    <li>python312Packages.dbt-common.dist</li>
    <li>python312Packages.dbt-postgres</li>
    <li>python312Packages.dbt-postgres.dist</li>
    <li>python312Packages.dbt-redshift</li>
    <li>python312Packages.dbt-redshift.dist</li>
    <li>python312Packages.dbt-snowflake</li>
    <li>python312Packages.dbt-snowflake.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.napalm</li>
    <li>python312Packages.napalm-hp-procurve</li>
    <li>python312Packages.napalm-hp-procurve.dist</li>
    <li>python312Packages.napalm.dist</li>
    <li>python312Packages.pyatv</li>
    <li>python312Packages.pyatv.dist</li>
    <li>python312Packages.pylitterbot</li>
    <li>python312Packages.pylitterbot.dist</li>
    <li>python312Packages.pytest-ansible</li>
    <li>python312Packages.pytest-ansible.dist</li>
    <li>python312Packages.pytest-testinfra</li>
    <li>python312Packages.pytest-testinfra.dist</li>
    <li>ttp (python312Packages.ttp)</li>
    <li>ttp.dist (python312Packages.ttp.dist)</li>
    <li>sherlock</li>
    <li>sherlock.dist</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>appdaemon</li>
    <li>appdaemon.dist</li>
    <li>deepdiff (python312Packages.deepdiff)</li>
    <li>deepdiff.dist (python312Packages.deepdiff.dist)</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.atsim-potentials</li>
    <li>python311Packages.atsim-potentials.dist</li>
    <li>python311Packages.deepdiff</li>
    <li>python311Packages.deepdiff.dist</li>
    <li>python311Packages.orderly-set</li>
    <li>python311Packages.orderly-set.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python312Packages.albumentations</li>
    <li>python312Packages.albumentations.dist</li>
    <li>python312Packages.atsim-potentials</li>
    <li>python312Packages.atsim-potentials.dist</li>
    <li>python312Packages.orderly-set</li>
    <li>python312Packages.orderly-set.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
  </ul>
</details>

natsukium commented on pkgs/development/python-modules/orderly-set/default.nix:
```diff
@@ -0,0 +1,45 @@
+  ];
+  nativeCheckInputs = [
+    pytestCheckHook
+    mypy
```
```suggestion
```
For the same reason, there is no need for static analysis.

doronbehar replied:
> For the same reason, there is no need for static analysis.

Same with this.

doronbehar replied:
> > For the same reason, there is no need for static analysis.
> 
> Same with this.

Found a way - simply disabled the relevant tests..

doronbehar commented on pkgs/development/python-modules/deepdiff/default.nix:
```diff
@@ -56,6 +67,9 @@ buildPythonPackage rec {
     tomli-w
+    polars
+    pandas
+    pytest-benchmark
```
> We don't want to instrument benchmarks in check phase. It will only slow down the build server.

I too wanted to avoid that, but I don't know how to make the build phase pass without this :/.

doronbehar replied:
> > We don't want to instrument benchmarks in check phase. It will only slow down the build server.
> 
> I too wanted to avoid that, but I don't know how to make the build phase pass without this :/.

Same here - disabled the relevant tests.

doronbehar committed:
python312Packages.dbt-common: reorder function inputs

doronbehar committed:
python312Packages.dbt-common: 1.8.0 -> 1.11.0

Not yet fully building, but with a Python package set that uses
protobuf4 instead of protobuf5, it is working.

doronbehar committed:
python311Packages.ttp: disable more failing tests

doronbehar commented:
I fixed (hopefully all) direct dependents of deepdiff. Let's how smaller the nixpkgs-review failure list will be now.

doronbehar commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 348567`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>azure-cli-extensions.confcom</li>
    <li>azure-cli-extensions.confcom.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 24 packages failed to build:</summary>
  <ul>
    <li>dbt (python312Packages.dbt-core)</li>
    <li>dbt.dist (python312Packages.dbt-core.dist)</li>
    <li>python311Packages.dbt-adapters</li>
    <li>python311Packages.dbt-adapters.dist</li>
    <li>python311Packages.dbt-bigquery</li>
    <li>python311Packages.dbt-bigquery.dist</li>
    <li>python311Packages.dbt-core</li>
    <li>python311Packages.dbt-core.dist</li>
    <li>python311Packages.dbt-postgres</li>
    <li>python311Packages.dbt-postgres.dist</li>
    <li>python311Packages.dbt-redshift</li>
    <li>python311Packages.dbt-redshift.dist</li>
    <li>python311Packages.dbt-snowflake</li>
    <li>python311Packages.dbt-snowflake.dist</li>
    <li>python312Packages.dbt-adapters</li>
    <li>python312Packages.dbt-adapters.dist</li>
    <li>python312Packages.dbt-bigquery</li>
    <li>python312Packages.dbt-bigquery.dist</li>
    <li>python312Packages.dbt-postgres</li>
    <li>python312Packages.dbt-postgres.dist</li>
    <li>python312Packages.dbt-redshift</li>
    <li>python312Packages.dbt-redshift.dist</li>
    <li>python312Packages.dbt-snowflake</li>
    <li>python312Packages.dbt-snowflake.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 117 packages built:</summary>
  <ul>
    <li>ansible (ansible_2_17 ,python312Packages.ansible-core)</li>
    <li>ansible-later</li>
    <li>ansible-later.dist</li>
    <li>ansible-lint</li>
    <li>ansible-lint.dist</li>
    <li>ansible-navigator</li>
    <li>ansible-navigator.dist</li>
    <li>ansible.dist (ansible_2_17.dist ,python312Packages.ansible-core.dist)</li>
    <li>ansible_2_15</li>
    <li>ansible_2_15.dist</li>
    <li>ansible_2_16</li>
    <li>ansible_2_16.dist</li>
    <li>appdaemon</li>
    <li>appdaemon.dist</li>
    <li>deepdiff (python312Packages.deepdiff)</li>
    <li>deepdiff.dist (python312Packages.deepdiff.dist)</li>
    <li>home-assistant-component-tests.apple_tv</li>
    <li>home-assistant-component-tests.litterrobot</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>molecule (python312Packages.molecule)</li>
    <li>molecule.dist (python312Packages.molecule.dist)</li>
    <li>napalm</li>
    <li>napalm.dist</li>
    <li>peering-manager</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.ansible</li>
    <li>python311Packages.ansible-compat</li>
    <li>python311Packages.ansible-compat.dist</li>
    <li>python311Packages.ansible-core</li>
    <li>python311Packages.ansible-core.dist</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-kernel.dist</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.ansible-runner.dist</li>
    <li>python311Packages.ansible-vault-rw</li>
    <li>python311Packages.ansible-vault-rw.dist</li>
    <li>python311Packages.ansible.dist</li>
    <li>python311Packages.atsim-potentials</li>
    <li>python311Packages.atsim-potentials.dist</li>
    <li>python311Packages.dbt-common</li>
    <li>python311Packages.dbt-common.dist</li>
    <li>python311Packages.deepdiff</li>
    <li>python311Packages.deepdiff.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.molecule</li>
    <li>python311Packages.molecule.dist</li>
    <li>python311Packages.napalm</li>
    <li>python311Packages.napalm-hp-procurve</li>
    <li>python311Packages.napalm-hp-procurve.dist</li>
    <li>python311Packages.napalm.dist</li>
    <li>python311Packages.orderly-set</li>
    <li>python311Packages.orderly-set.dist</li>
    <li>python311Packages.pyatv</li>
    <li>python311Packages.pyatv.dist</li>
    <li>python311Packages.pylitterbot</li>
    <li>python311Packages.pylitterbot.dist</li>
    <li>python311Packages.pytest-ansible</li>
    <li>python311Packages.pytest-ansible.dist</li>
    <li>python311Packages.pytest-testinfra</li>
    <li>python311Packages.pytest-testinfra.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.ttp</li>
    <li>python311Packages.ttp.dist</li>
    <li>python312Packages.albumentations</li>
    <li>python312Packages.albumentations.dist</li>
    <li>python312Packages.ansible</li>
    <li>python312Packages.ansible-compat</li>
    <li>python312Packages.ansible-compat.dist</li>
    <li>python312Packages.ansible-kernel</li>
    <li>python312Packages.ansible-kernel.dist</li>
    <li>python312Packages.ansible-runner</li>
    <li>python312Packages.ansible-runner.dist</li>
    <li>python312Packages.ansible-vault-rw</li>
    <li>python312Packages.ansible-vault-rw.dist</li>
    <li>python312Packages.ansible.dist</li>
    <li>python312Packages.atsim-potentials</li>
    <li>python312Packages.atsim-potentials.dist</li>
    <li>python312Packages.dbt-common</li>
    <li>python312Packages.dbt-common.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.napalm</li>
    <li>python312Packages.napalm-hp-procurve</li>
    <li>python312Packages.napalm-hp-procurve.dist</li>
    <li>python312Packages.napalm.dist</li>
    <li>python312Packages.orderly-set</li>
    <li>python312Packages.orderly-set.dist</li>
    <li>python312Packages.pyatv</li>
    <li>python312Packages.pyatv.dist</li>
    <li>python312Packages.pylitterbot</li>
    <li>python312Packages.pylitterbot.dist</li>
    <li>python312Packages.pytest-ansible</li>
    <li>python312Packages.pytest-ansible.dist</li>
    <li>python312Packages.pytest-testinfra</li>
    <li>python312Packages.pytest-testinfra.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
    <li>ttp (python312Packages.ttp)</li>
    <li>ttp.dist (python312Packages.ttp.dist)</li>
    <li>sherlock</li>
    <li>sherlock.dist</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
  </ul>
</details>

doronbehar commented:
OK Great the dbt failures are not new failure, e.g:

https://hydra.nixos.org/build/274882178

I'm not going to fix that as it is unrelated to this update. The maintainers of these packages will become aware of these failures at some point. Will let ofborg ping the maintainers and will wait a day or two, and will merge afterwards if no objections arise.

doronbehar commented:
Rebased after a merge conflict (it seems someone has fixed an issue with `python312Packages.dbt-common`).

dotlambda commented:
We should be able to get rid of numpy by updating deepdiff to 8.0.1: https://github.com/seperman/deepdiff/pull/479

doronbehar commented:
OK, upstream is communicative (e.g https://github.com/seperman/deepdiff/pull/496 ).

dotlambda commented on pkgs/development/python-modules/deepdiff/default.nix:
```diff
@@ -2,44 +2,60 @@
   pname = "deepdiff";
-  version = "7.0.1";
-  format = "setuptools";
+  version = "8.0.0";
```
```suggestion
  version = "8.0.1";
```

dotlambda commented on pkgs/development/python-modules/deepdiff/default.nix:
```diff
@@ -2,44 +2,60 @@
   src = fetchFromGitHub {
     owner = "seperman";
     repo = "deepdiff";
     rev = "refs/tags/${version}";
```
```suggestion
    rev = "6d8a4c7c32d5ac57919955954790be994d01fe57";
```
until https://github.com/seperman/deepdiff/issues/497 is resolved

doronbehar replied:
> until [seperman/deepdiff#497](https://github.com/seperman/deepdiff/issues/497) is resolved

Good idea, done :).

doronbehar committed:
python312Packages.deepdiff: 8.0.0 -> 8.0.1

doronbehar commented:
Also removed the `numpy` related fixes to the direct dependencies. Tested most of the dependent packages, and fixed `dbt-common` which is failing on Hydra currently (no maintainers).

doronbehar commented:
Fixed the 8.0.1 tagging issue after upstream fixed it.

dotlambda commented:
Result of `nixpkgs-review pr 348567` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>141 packages failed to build:</summary>
  <ul>
    <li>ansible (python312Packages.ansible-core)</li>
    <li>ansible-later</li>
    <li>ansible-later.dist</li>
    <li>ansible-lint</li>
    <li>ansible-lint.dist</li>
    <li>ansible-navigator</li>
    <li>ansible-navigator.dist</li>
    <li>ansible.dist (python312Packages.ansible-core.dist)</li>
    <li>ansible_2_15</li>
    <li>ansible_2_15.dist</li>
    <li>ansible_2_16</li>
    <li>ansible_2_16.dist</li>
    <li>appdaemon</li>
    <li>appdaemon.dist</li>
    <li>dbt (python312Packages.dbt-core)</li>
    <li>dbt.dist (python312Packages.dbt-core.dist)</li>
    <li>deepdiff (python312Packages.deepdiff)</li>
    <li>deepdiff.dist (python312Packages.deepdiff.dist)</li>
    <li>home-assistant-component-tests.apple_tv</li>
    <li>home-assistant-component-tests.litterrobot</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>molecule (python312Packages.molecule)</li>
    <li>molecule.dist (python312Packages.molecule.dist)</li>
    <li>napalm</li>
    <li>napalm.dist</li>
    <li>peering-manager</li>
    <li>poetry</li>
    <li>poetry.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>poetryPlugins.poetry-plugin-up</li>
    <li>poetryPlugins.poetry-plugin-up.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.ansible</li>
    <li>python311Packages.ansible-compat</li>
    <li>python311Packages.ansible-compat.dist</li>
    <li>python311Packages.ansible-core</li>
    <li>python311Packages.ansible-core.dist</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-kernel.dist</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.ansible-runner.dist</li>
    <li>python311Packages.ansible-vault-rw</li>
    <li>python311Packages.ansible-vault-rw.dist</li>
    <li>python311Packages.ansible.dist</li>
    <li>python311Packages.atsim-potentials</li>
    <li>python311Packages.atsim-potentials.dist</li>
    <li>python311Packages.dbt-adapters</li>
    <li>python311Packages.dbt-adapters.dist</li>
    <li>python311Packages.dbt-bigquery</li>
    <li>python311Packages.dbt-bigquery.dist</li>
    <li>python311Packages.dbt-common</li>
    <li>python311Packages.dbt-common.dist</li>
    <li>python311Packages.dbt-core</li>
    <li>python311Packages.dbt-core.dist</li>
    <li>python311Packages.dbt-postgres</li>
    <li>python311Packages.dbt-postgres.dist</li>
    <li>python311Packages.dbt-redshift</li>
    <li>python311Packages.dbt-redshift.dist</li>
    <li>python311Packages.dbt-snowflake</li>
    <li>python311Packages.dbt-snowflake.dist</li>
    <li>python311Packages.deepdiff</li>
    <li>python311Packages.deepdiff.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.molecule</li>
    <li>python311Packages.molecule.dist</li>
    <li>python311Packages.napalm</li>
    <li>python311Packages.napalm-hp-procurve</li>
    <li>python311Packages.napalm-hp-procurve.dist</li>
    <li>python311Packages.napalm.dist</li>
    <li>python311Packages.orderly-set</li>
    <li>python311Packages.orderly-set.dist</li>
    <li>python311Packages.pyatv</li>
    <li>python311Packages.pyatv.dist</li>
    <li>python311Packages.pylitterbot</li>
    <li>python311Packages.pylitterbot.dist</li>
    <li>python311Packages.pytest-ansible</li>
    <li>python311Packages.pytest-ansible.dist</li>
    <li>python311Packages.pytest-testinfra</li>
    <li>python311Packages.pytest-testinfra.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.ttp</li>
    <li>python311Packages.ttp.dist</li>
    <li>python312Packages.albumentations</li>
    <li>python312Packages.albumentations.dist</li>
    <li>python312Packages.ansible</li>
    <li>python312Packages.ansible-compat</li>
    <li>python312Packages.ansible-compat.dist</li>
    <li>python312Packages.ansible-kernel</li>
    <li>python312Packages.ansible-kernel.dist</li>
    <li>python312Packages.ansible-runner</li>
    <li>python312Packages.ansible-runner.dist</li>
    <li>python312Packages.ansible-vault-rw</li>
    <li>python312Packages.ansible-vault-rw.dist</li>
    <li>python312Packages.ansible.dist</li>
    <li>python312Packages.atsim-potentials</li>
    <li>python312Packages.atsim-potentials.dist</li>
    <li>python312Packages.dbt-adapters</li>
    <li>python312Packages.dbt-adapters.dist</li>
    <li>python312Packages.dbt-bigquery</li>
    <li>python312Packages.dbt-bigquery.dist</li>
    <li>python312Packages.dbt-common</li>
    <li>python312Packages.dbt-common.dist</li>
    <li>python312Packages.dbt-postgres</li>
    <li>python312Packages.dbt-postgres.dist</li>
    <li>python312Packages.dbt-redshift</li>
    <li>python312Packages.dbt-redshift.dist</li>
    <li>python312Packages.dbt-snowflake</li>
    <li>python312Packages.dbt-snowflake.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.napalm</li>
    <li>python312Packages.napalm-hp-procurve</li>
    <li>python312Packages.napalm-hp-procurve.dist</li>
    <li>python312Packages.napalm.dist</li>
    <li>python312Packages.orderly-set</li>
    <li>python312Packages.orderly-set.dist</li>
    <li>python312Packages.pyatv</li>
    <li>python312Packages.pyatv.dist</li>
    <li>python312Packages.pylitterbot</li>
    <li>python312Packages.pylitterbot.dist</li>
    <li>python312Packages.pytest-ansible</li>
    <li>python312Packages.pytest-ansible.dist</li>
    <li>python312Packages.pytest-testinfra</li>
    <li>python312Packages.pytest-testinfra.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
    <li>ttp (python312Packages.ttp)</li>
    <li>ttp.dist (python312Packages.ttp.dist)</li>
    <li>sherlock</li>
    <li>sherlock.dist</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
  </ul>
</details>
